tox.ini

```diff
@@ -4,9 +4,25 @@ envlist = py27
 [testenv]
 deps = pytest
 install_command = pip install -e .[test] --find-links {toxinidir}/3rdparty {opts} {packages}
-# TODO(xujyan): Fix up 'test_scheduler_runs'.
 commands = py.test -v -k 'not test_scheduler_runs' tests {posargs:}
 
+[testenv:pex]
+deps =
+  pex
+  pytest
+  wheel
+whitelist_externals=mkdir
+commands =
+  mkdir -p {toxinidir}/dist/
+  pip install --find-links {toxinidir}/3rdparty -e .[driver]
```

Comment:
I am trying not to require `mesos.native` in the main package so people don't need to download/build one just to run tests or otherwise get started. 

Instead I am listing `mesos.native` [as an extra](https://github.com/twitter/mysos/blob/jyx/pex/setup.py#L64):
```
'driver': ['mesos.native{0}'.format(MESOS_VERSION),],
```
and having the binaries use the extra.
```
'mysos_scheduler=mysos.scheduler.mysos_scheduler:proxy_main [driver]',
```

Therefore in the integration test that does require the native driver, I need `pip install` to include the extra and look for `mesos.native` that I dropped in in the `3rdparty` folder, which I don't think tox knows to do it directly so I ended up writing the custom `pip install` command here.

The reason for 
```
mkdir -p {toxinidir}/dist/
```
is that I need a place to put the generated pex and have my test find it in that `dist` folder.

FWIW I excluded this test from regular tests so the CI don't need to bother to install `mesos.native`.